IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Types/IfcGeotechTypicalSectionTypeEnum/Documentation.md

```diff
@@ -0,0 +1 @@
+This container defines the different predefined types of geotech typical sections that can specify an _IfcGeotechTypicalSection_.
```

Comment:
"geotech typical sections" should become "typical geotechnical sections"

Reply:
Agreed.

Reply:
Done.